specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,214 @@
+                     }
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "name":"scope",
+                  "in":"path",
+                  "description":"Uri path corresponding to commercial subscription or legacy subscription",
+                  "required":true,
+                  "type":"string"
+               },
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               },
+               {  
+                  "name":"body",
+                  "in":"body",
+                  "description":"The subscription creation request object.",
+                  "required":true,
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationRequest"
+                  }
+               }
+            ]
+         }
+      }
+   },
+   "definitions":{  
+      "SubscriptionCreationResult":{  
+         "properties":{  
+            "subscriptionId":{  
+               "type":"string"
+            }
+         }
+      },
+      "SubscriptionCreationRequest":{  
+         "properties":{  
+            "subscriptionDisplayName":{  
+               "type":"string"
+            },
+            "Owners":{  
+               "type":"array",
+               "items":{  
+                  "$ref":"#/definitions/AdPrincipal"
+               },
+               "description":"An array of Active Directory Principals"
+            },
+            "offerId":{  
+               "type":"string"
+            }
+         }
+      },
+      "ErrorResponse":{  
+         "description":"Describes the format of Error response.",
+         "type":"object",
+         "properties":{  
+            "code":{  
+               "description":"Error code",
+               "type":"string"
+            },
+            "message":{  
+               "description":"Error message indicating why the operation failed.",
+               "type":"string"
+            }
+         }
+      },
+      "AdPrincipal":{  
+         "description":"Active Directory Principal for subscription creation delegated permission",
+         "properties":{  
+            "tenantId":{  
+               "description":"Tenant id of the Principal",
+               "type":"string"
+            },
+            "objectId":{  
+               "description":"Object id of the Principal",
+               "type":"string"
+            }
+         }
+      },
+      "OperationListResult":{  
+         "description":"status of the subscription POST operation.",
+         "properties":{  
+            "pendingOperations":{  
+               "type":"array",
+               "items":{  
+                  "$ref":"#/definitions/SubscriptionOperation"
+               }
+            }
+         }
+      },
+      "SubscriptionOperation":{  
+         "description":"status of the subscription POST operation.",
+         "properties":{  
+            "Location":{  
```

Comment:
These only make sense in the context of headers. When someone gets a list of subscription operations, each subscription operation object should simply have a property such as "status": "<string>" and maybe "creationDate": "<date>".